Description: further fixes for building with GCC 13
 Followup to 10fa4ebe6b2868460c845b524c0379b8dc216cfc.
 All of these seem to have landed for Chromium 114 upstream.
Origin: upstream, https://code.qt.io/cgit/qt/qtwebengine-chromium.git/commit/?id=8da9e3d7bea3286e
Last-Update: 2023-06-19

--- a/src/3rdparty/chromium/device/base/synchronization/one_writer_seqlock.cc
+++ b/src/3rdparty/chromium/device/base/synchronization/one_writer_seqlock.cc
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include <cstdint>
+
 #include "device/base/synchronization/one_writer_seqlock.h"
 
 namespace device {
--- a/src/3rdparty/chromium/gpu/command_buffer/common/webgpu_cmd_format.h
+++ b/src/3rdparty/chromium/gpu/command_buffer/common/webgpu_cmd_format.h
@@ -5,6 +5,7 @@
 #ifndef GPU_COMMAND_BUFFER_COMMON_WEBGPU_CMD_FORMAT_H_
 #define GPU_COMMAND_BUFFER_COMMON_WEBGPU_CMD_FORMAT_H_
 
+#include <cstdint>
 #include <string.h>
 
 #include "gpu/command_buffer/common/gl2_types.h"
--- a/src/3rdparty/chromium/gpu/command_buffer/service/shared_context_state.h
+++ b/src/3rdparty/chromium/gpu/command_buffer/service/shared_context_state.h
@@ -5,6 +5,7 @@
 #ifndef GPU_COMMAND_BUFFER_SERVICE_SHARED_CONTEXT_STATE_H_
 #define GPU_COMMAND_BUFFER_SERVICE_SHARED_CONTEXT_STATE_H_
 
+#include <cstdint>
 #include <memory>
 #include <vector>
 
--- a/src/3rdparty/chromium/gpu/config/gpu_util.h
+++ b/src/3rdparty/chromium/gpu/config/gpu_util.h
@@ -5,6 +5,8 @@
 #ifndef GPU_CONFIG_GPU_UTIL_H_
 #define GPU_CONFIG_GPU_UTIL_H_
 
+#include <cstdint>
+
 #include "build/build_config.h"
 #include "gpu/config/gpu_feature_info.h"
 #include "gpu/gpu_export.h"
--- a/src/3rdparty/chromium/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc
+++ b/src/3rdparty/chromium/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc
@@ -4,6 +4,7 @@
 
 #include "third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.h"
 
+#include <cstdint>
 #include <set>
 
 #include "base/memory/singleton.h"
--- a/src/3rdparty/chromium/third_party/libgav1/src/src/utils/threadpool.cc
+++ b/src/3rdparty/chromium/third_party/libgav1/src/src/utils/threadpool.cc
@@ -28,6 +28,7 @@
 #include <cassert>
 #include <cinttypes>
 #include <cstddef>
+#include <cstdio>
 #include <cstdint>
 #include <cstring>
 #include <new>
--- a/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/importers/proto/proto_importer_module.h
+++ b/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/importers/proto/proto_importer_module.h
@@ -17,6 +17,8 @@
 #ifndef SRC_TRACE_PROCESSOR_IMPORTERS_PROTO_PROTO_IMPORTER_MODULE_H_
 #define SRC_TRACE_PROCESSOR_IMPORTERS_PROTO_PROTO_IMPORTER_MODULE_H_
 
+#include <cstdint>
+
 #include "perfetto/ext/base/optional.h"
 #include "perfetto/trace_processor/status.h"
 #include "src/trace_processor/trace_blob_view.h"
--- a/src/3rdparty/chromium/v8/src/base/macros.h
+++ b/src/3rdparty/chromium/v8/src/base/macros.h
@@ -5,6 +5,7 @@
 #ifndef V8_BASE_MACROS_H_
 #define V8_BASE_MACROS_H_
 
+#include <cstdint>
 #include <limits>
 #include <type_traits>
